### DISCUSSION TITLE
How to create a job that targets assets in different code locations

### DISCUSSION THREAD

**Discussion by clairelin135** [dagster-io/dagster#16847] [Q&A]
I want to schedule a job that targets assets in different code locations. Is this possible?

**Comment by clairelin135** [comment]
Unfortunately this is not possible, the main reason is that all steps in a job will be executed in one same environment, whereas different repos (or code locations) can have different python execution environments. So a single job can't execute cross-code location.

One workaround would be to use an [asset sensor](https://docs.dagster.io/concepts/partitions-schedules-sensors/asset-sensors#defining-an-asset-sensor) which can function cross repository. The asset sensor can exist in the downstream code location and monitor when the assets in the upstream code location materialize, then kick off a run for the downstream assets.

**Comment by axellpadilla** [comment]
@clairelin135 Automation Conditions is the answer for this now right?
